nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/groups/StandardProcessGroup.java

```diff
@@ -4422,6 +4426,26 @@ public QueueSize getQueueSize() {
         return new QueueSize(count, contentSize);
     }
 
+    @Override
+    public Boolean isLogToOwnFile() {
+        return logToOwnFile;
+    }
+
+    @Override
+    public void setLogToOwnFile(final Boolean logToOwnFile) {
+        this.logToOwnFile = logToOwnFile;
+    }
+
+    @Override
+    public String getLogFileSuffix() {
+        return logFileSuffix;
+    }
+
+    @Override
+    public void setLogFileSuffix(final String logFileSuffix) {
+        this.logFileSuffix = logFileSuffix;
```

Comment:
We should disallow any characters that are not filename friendly. For example, we should not allow spaces, slashes (`/`) or backslashes (`\`)